cli-plugins/manager/hooks_test.go

```diff
@@ -36,3 +36,43 @@ func TestGetNaiveFlags(t *testing.T) {
 		assert.DeepEqual(t, getNaiveFlags(tc.args), tc.expectedFlags)
 	}
 }
+
+func TestPluginMatch(t *testing.T) {
+	testCases := []struct {
+		commandString string
+		pluginConfig  map[string]string
+		expectedMatch string
+		expectedOk    bool
+	}{
+		{
+			commandString: "image ls",
+			pluginConfig: map[string]string{
+				"hooks": "image",
```

Comment:
do you mind adding some test cases where "hooks" contains multiple comma separated values, so we're sure we go through that code path?

Reply:
also, the ordering of the comma separated values is important since the first commandString prefix match will determine the returned value, so i think we should test that as well. 

eg. if `"hooks": "image ls,image"` i'd expect `expectedMatch: "image ls"`, does that sound right to you?